tests/testthat/test-linter_tags.R

```diff
@@ -13,7 +13,11 @@ test_that("validate_linter_db works as expected", {
   )
   expect_false(suppressWarnings(lintr:::validate_linter_db(df_empty, "mypkg")))
 
-  df <- data.frame(linter = "absolute_path_linter", tags = "robustness")
+  df <- data.frame(
```

Comment:
we should rather get rid of this as part of #1692 :)

Reply:
True, but maybe still worth including this change until that issue is closed?

Reply:
agree